### ISSUE TITLE
Right Click on any engine Crashes Game /server in SP or MP no optifine

### ISSUE THREAD

**Issue by Tekkwiz32** [BuildCraft/BuildCraft#4231]
BuildCraft version: 7.99.16
Forge version: 1.12.2-14.23.1.2554
Link to crash report or log: https://pastebin.com/Epy8GF0S
Singleplayer or multiplayer: both
Steps to reproduce: Right click on engine 
Additional information: When Right click on engine Single or multiplayer crashes game and or server. Optifine is not installed

**Comment by Tekkwiz32** [comment]
Any one?

**Comment by AlexIIL** [comment]
Sorry, I probably won't get around to looking into this for a while. In addition I'm not sure how that even happened - something generated a null pointer from inside *vanilla* code, which is a bit odd.

**Comment by Tekkwiz32** [comment]
Okay so me being very inexperienced dove into this issues with the null pointer on vanilla code. So I found that my Minecraft launncher was the root of it.  It was going through a profile unknown to me then loading profiles over this as in my profile as to like saving the account. So I started all over I downloaded the game fresh from majong then reloaded the same forge files and the buildcraft mod . And scuccess I found later that my launcher was replaced by a version of mineshafter. If your having errors with build craft. Check to make sure your not using mineshafter first. Huge love and thanks for the buildcraft mod/add-on it has remade Minecraft for me..
